src/asm3/reports.py

```diff
@@ -732,6 +732,21 @@ def get_mailmerges_menu(dbo: Database, roleids: str = "", superuser: bool = Fals
             mv.append( ( asm3.users.MAIL_MERGE, "", "", "mailmerge?id=%d" % m.ID, "", m.TITLE ) )
     return mv
 
+def get_internalforms_menu(dbo: Database) -> MenuItems:
+    """
+    Reads the list of internal online forms and returns them as a list for inserting into
+    our menu structure.
+    The return value is a list of online forms with a tuple containing URL and
+    name.
+    roleids: comma separated list of roleids for the current user
+    superuser: true if the user is a superuser
+    """
+    mv = []
+    forms = dbo.query("SELECT ID, Name FROM onlineform WHERE InternalUse = 1 ORDER BY Name")
+    for form in forms:
+        mv.append( ( asm3.users.VIEW_ONLINE_FORMS, "", "", "/onlineform_view?formid=%d" % form.ID, "", form.NAME ) )
```

Comment:
leave off the leading forward slash - otherwise you'll break installations where the system isn't at the root of the domain (which is virtually all open source installs with an asm3 prefix!)